examples.sty

```diff
@@ -85,25 +87,31 @@
       { , }
       { #1 }
       {
-        \fontsize{30}{30}
-        \selectfont
-        \raisebox{-5.5pt}{\begin{tikzpicture}
-          \fill[white] (0.6,-0.35) rectangle (1.4,-1.35);
-          \node[inner~sep=0pt, outer~sep=0pt] at (1,-0.85) {\tl_use:N \l_tmpa_tl};
-          \fill[white] (0.9,-0.05) rectangle (1.7,-1.05);
-          \node[inner~sep=0pt, outer~sep=0pt] at (1.3,-0.55) {\tl_use:N \l_tmpa_tl};
-          \fill[white] (1.2,0.35) rectangle (2,-0.65);
-          \node[inner~sep=0pt, outer~sep=0pt] at (1.6,-0.15) {\tl_use:N \l_tmpa_tl};
+        \raisebox{-5.5pt}{\begin{tikzpicture}[every~node/.style={\l_tmpa_tl,fill=white,draw=black,}]
+          \node at (1,-0.85) {};
+          \node at (1.3,-0.55) {};
+          \node at (1.6,-0.15) {};
         \end{tikzpicture}}
         \vspace{0.05cm}
       }
       {
-        \fontsize{45}{45}
-        \selectfont
-        \tl_use:N \l_tmpa_tl
+        \icon_node:n {}
       }
     \group_end:
   }
+  
+\cs_set:Nn
+\large_icon:nn
+{
+  \icon:n
+    { #2 }
+       \begin{scope}[every~node/.style={\l_tmpa_tl,fill=white,draw=black,scale=.5},shift={(#1)},scale=.5]
+         \node at (1,-0.85) {};
+         \node at (1.3,-0.55) {};
+         \node at (1.6,-0.15) {};
+         \end{scope}
```

Comment:
There is a huge gap below the large icon and its label:

### Before
![image](https://github.com/Witiko/fast-regression-testing/assets/603082/51c1018e-176f-4376-8b5a-88cf50aeab06)

### After
![image](https://github.com/Witiko/fast-regression-testing/assets/603082/9f94750b-56bc-4ecd-9dff-637e3347f670)

Reply:
yup i know wanted to know first if you generally are okay wih the icons, will adjust right now